.github/workflows/pyfiction-pypi-deployment.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           - { os: ubuntu-latest, arch: x64 }
           - { os: macos-latest, arch: x64 }
           - { os: macos-latest, arch: arm64 }
```

Comment:
Is there a reason for not specifying the exact versions here, too?

Reply:
No, I will update this part.